vtex-apps/product-customizer
# Make input values work for recursive assemblies

**What problem is this solving?**

Make Input Values work on recursive assemblies.

**How should this be manually tested?**

1. Open https://breno--storecomponents.myvtex.com/custom-bell/p
2. Click on Customize
3. A modal will open, click on the button "Testing"
4. Type anything in the fields
5. Click "DONE"
6. Click on Customize again
7. Check if the fields are filled with the inputs of step 4
